hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -456,4 +456,15 @@ private Constants() {
   public static final String FAIL_INJECT_INCONSISTENCY_PROBABILITY =
       "fs.s3a.failinject.inconsistency.probability";
 
+  public static final String MULTIPART_DOWNLOAD_PART_SIZE = "fs.s3a.multipartdownload.part-size";
```

Comment:
nit: might want to indicate unit (`bytes`?) in key

Reply:
If you retrieve the value using `Configuration.getLongBytes()` you get specify values like "64M", "4T